internal/handlers/common/params.go

```diff
@@ -363,11 +369,11 @@ func addNumbers(v1, v2 any) (any, error) {
 		case int64:
 			if v2 > 0 {
 				if int64(v1) > math.MaxInt64-v2 {
-					return nil, errLongExceeded
+					return nil, errLongExceededPositive
 				}
 			} else {
 				if int64(v1) < math.MinInt64-v2 {
-					return nil, errLongExceeded
+					return nil, errLongExceededPositive
```

Comment:
Missed this first way around but I think it is `errLongExceededNegative`. But this doesn't really have any implication for addNumbers, since whichever error produce the same error message+code. Anyway something to address in upcoming PR.

Reply:
Nice catch! Let me fix it in this PR!